buildSrc/build.gradle.kts

```diff
@@ -30,6 +32,8 @@ project.tasks.withType(JavaCompile::class.java).configureEach {
 // ensure the Kotlin + Java compilers both use the same language level.
 project.tasks.withType(KotlinCompile::class.java).configureEach {
     kotlinOptions {
-        jvmTarget = "1.8"
+        apiVersion = "1.7"
+        languageVersion = "1.7"
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
I don't think this is strictly necessary as it applies to tasks in the build itself, although we could keep it if it doesn't cause any problems

Reply:
If it's not necessary I don't really want to add extra changes. I'll remove it